src/main/scala/spark/dicom/v2/DicomDataSource.scala

```diff
@@ -54,17 +57,59 @@ object DicomDataSource {
             nullable = true
           )
       }
-    val selectedFields =
-      if (withPixelData) fields
-      else fields.filter(p => p.name != Keyword.valueOf(Tag.PixelData))
+    fields
+  }
+
+  def schema(
+      includePixelData: Boolean,
+      includeContent: Boolean,
+      includePrivateTags: Boolean
```

Comment:
Would it be useful to have default values for these bools?

Reply:
Also, from the end user point of view, these parameters do have a default value (false).